commands/user_e2e_test.go

```diff
@@ -559,6 +559,7 @@ func (s *MmctlE2ETestSuite) TestUpdateUsernameCmd() {
 		printer.Clean()
 		newUsername := "basicusernamechange"
 		err := updateUsernameCmdF(s.th.Client, &cobra.Command{}, []string{s.th.BasicUser2.Id, newUsername})
+		s.Require().NotNil(err)
```

Comment:
This check is redundant, `EqualError` already checks if there is an error.

Reply:
Good catch, thanks